src/js/services/lang.js

```diff
@@ -51,6 +51,7 @@ const setLanguageSelectorOnLoad = () => {
     languages[window.navigator.language] ||
     languages['en'];
   $(currentLang.domEl).css('background', '#F3F3F3');
+  $(currentLang.domEl).css('font-weight', 'bold');
```

Comment:
I think it would be better if a selected language had a dedicated CSS class and we just toggle it depending on if it's selected or not. Easier to maintain in the long run